### PR TITLE
FE Test Library Timeout

### PR DESCRIPTION
**Pull request by Siolto**
Fiori Elements TestLib will use the same `timeouts` as defined in our property `waitForUI5Timeout`. There is also a new hint in the documentation which tells, that every timeout higher than 30 seconds requires to increase the [session script timeout](https://webdriver.io/docs/timeouts/#session-script-timeout) as well.

### CODE REVIEW

File: client-side-js/testLibrary.js [hunk 1]
```diff
@@ -10,7 +10,13 @@ async function initOPA(pageObjectConfig, browserInstance) {
                         pageConfig[pageKey] = new window.fe_bridge[className](options)
                     })
                 })
+
                 sap.ui.test.Opa5.createPageObjects(pageConfig)
+                // use the same timouts and intervals that wdi5 uses
+                sap.ui.test.Opa.extendConfig({
+                    timeout: new Date(window.wdi5.waitForUI5Options.timeout).getSeconds(), // convert milliseconds to seconds
```
Comment (vobu) [inline client-side-js/testLibrary.js:17]: can `window.wdi5.waitForUI5Options.timeout` be `undefined`?

File: client-side-js/testLibrary.js [hunk 2]
```diff
@@ -10,7 +10,13 @@ async function initOPA(pageObjectConfig, browserInstance) {
...
+                    pollingInterval: window.wdi5.waitForUI5Options.interval
```
Comment (vobu) [inline client-side-js/testLibrary.js:18]: same q as above - might there be a potential runtime error for an `undefined` config option?

File: docs/configuration.md [hunk 3]
```diff
@@ -119,6 +119,8 @@ See <https://github.com/ui5-community/wdi5/tree/main/examples/ui5-js-app/webapp/
 
 Number in milliseconds (default: `15000`) to wait for UI5-related operations within a UI5 app. This is the equivalent to OPA5's [`waitFor()` option `timeout`](https://ui5.sap.com/sdk/#/api/sap.ui.test.Opa5/methods/waitFor).
 
+?> When setting this timeout to 30 seconds or higher requires increasing the [session script timeout](https://webdriver.io/docs/timeouts/#session-script-timeout) as well.
```
Comment (vobu) [inline docs/configuration.md:122]: is „When“ here superfluous?

File: client-side-js/testLibrary.js (hunk 1 above)
Comment (Siolto) [reply client-side-js/testLibrary.js:17]: In the [injectUi5](https://github.com/ui5-community/wdi5/blob/main/client-side-js/injectUI5.js#L23) script we make sure this property can not be undefined.

File: client-side-js/testLibrary.js (hunk 2 above)
Comment (Siolto) [reply client-side-js/testLibrary.js:18]: same answer as above

File: docs/configuration.md (hunk 3 above)
Comment (Siolto) [reply docs/configuration.md:122]:
```suggestion
?> Setting this timeout to 30 seconds or higher requires the [session script timeout](https://webdriver.io/docs/timeouts/#session-script-timeout) to be increased as well.
```

**Review by vobu — APPROVED** [review]
LGTM! Let's get through CI, then good to go :)

**Pull request merged**